py/trtorch/csrc/trtorch_py.cpp

```diff
@@ -196,8 +196,10 @@ PYBIND11_MODULE(_C, m) {
     .def_readwrite("refit",                &ExtraInfo::refit)
     .def_readwrite("debug",                &ExtraInfo::debug)
     .def_readwrite("strict_types",         &ExtraInfo::strict_types)
-    .def_readwrite("allow_gpu_fallback",   &ExtraInfo::allow_gpu_fallback)
-    .def_readwrite("device",               &ExtraInfo::device)
+    .def_readwrite("allow_gpu_fallback",   &ExtraInfo::device::allow_gpu_fallback)
```

Comment:
Is there  Python object that we use here to create a device? what is the API look like for the user?

Reply:
I added a struct to ExtraInfo
https://github.com/andi4191/TRTorch/blob/anuragd/dla_aarch64_support/py/trtorch/csrc/trtorch_py.cpp#L104

Reply:
How would this get used though? Can you add a test in the python_api_tests to demonstrate?

Reply:
ok